### PR TITLE
WRN-1351: Migrate ActionGuide and RadioItem unit tests to Testing Library

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated `ActionGuide` and `RadioItem` unit tests to react-testing-library

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-1351

### CODE REVIEW

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

File: RadioItem/tests/RadioItem-specs.js [hunk 1]
```diff
@@ -1,19 +1,61 @@
...
+
+		const actual = getByText('Hello RadioItem');
+
+		expect(actual).toBeInTheDocument();
+	});
+
+	test('should render a disabled RadioItem when `disabled` is true', () => {
+		const {getByRole} = render(<RadioItemBase disabled>Hello RadioItem</RadioItemBase>);
+
+		expect(getByRole('checkbox', {disabled: true}));
```
Comment (daniel-stoian-lgp) [inline RadioItem/tests/RadioItem-specs.js:29]: interesting approach :)

Comment (daniel-stoian-lgp) [reply RadioItem/tests/RadioItem-specs.js:29]: it seems to be a false positive expectation. if I put "disabled:false", the test still passes

Comment (adrian-cocoara-lgp) [reply RadioItem/tests/RadioItem-specs.js:29]: You're right, I thought it would behave like Checkbox. Thanks for suggestion

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Pull request merged**